.gitignore

```diff
@@ -2,3 +2,4 @@
 **/obj
 **/bin
 **/cobertura.xml
+.idea
```

Comment:
what is an .idea file?

Reply:
It's the IDE workspace directory created by the IDE by JetBrains ([IntelliJ IDEA](https://www.jetbrains.com/idea/)). I was using that at first before swapping to a vscode dev container so I could run the tests and stuff.

Reply:
I can remove it if we don't want it. I just wanted to prevent my IDE files from getting committed.

Reply:
Was just curious...seems fine to leave in :)